build.gradle

```diff
@@ -31,27 +37,55 @@ repositories {
 		name 'Shedaniel'
 		url 'https://maven.shedaniel.me'
 	}
+	maven {
+		name 'Sinytra'
+		url "https://maven.su5ed.dev/releases"
+	}
 }
 
 dependencies {
 	// To change the versions see the gradle.properties file
-	minecraft "com.mojang:minecraft:${project.minecraft_version}"
-	mappings "net.fabricmc:yarn:${project.yarn_mappings}:v2"
-	modImplementation "net.fabricmc:fabric-loader:${project.loader_version}"
-
-	// Fabric API
-	modImplementation "net.fabricmc.fabric-api:fabric-api:${project.fabric_version}"
+	minecraft "net.minecraft:minecraft:$project.minecraft_version"
+	mappings loom.layered {
+		it.mappings("net.fabricmc:yarn:$project.yarn_mappings:v2")
+		it.mappings("dev.architectury:yarn-mappings-patch-neoforge:$project.yarn_mappings_patch_version")
+	}
+	neoForge "net.neoforged:neoforge:$project.neoforge_version"
 
-	modImplementation("com.terraformersmc:modmenu:${modmenu_version}") {
-		exclude group: 'net.fabricmc.fabric-api'
+	// Fetch the version of the fabric-api modules from the pom file
```

Comment:
This is just for the sake of JiJ-ing FFAPI, so that it can only JiJ the needed mods.
If it's instead preferred to let players supply FFAPI themselves, this can all be simplified to a modImplementation to the collective FFAPI dependency.